src/components/fileUpload.js

```diff
@@ -200,24 +377,12 @@
           {helperValue}
         </FormHelperText>
         {loading && B.triggerEvent('onLoad')}
```

Comment:
I know this is not your code, but this should be done in a `useEffect` hook. This is no JSX 😁 . Can you move this? :)